### PR TITLE
fix: modify typo

### PR DESCRIPTION
**Pull request by yuzhiquan**
- fix: modify typo

### CODE REVIEW

**Comment by xueweiz** [comment]
Thanks for noticing this and fixing this @yuzhiquan !
/ok-to-test
/lgtm
/approve

File: pkg/systemlogmonitor/logwatchers/filelog/log_watcher.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func NewSyslogWatcherOrDie(cfg types.WatcherConfig) types.LogWatcher {
 	}
 }
 
-// Make sure NewSyslogWathcer is types.WatcherCreateFunc.
```
Comment (xueweiz) [inline pkg/systemlogmonitor/logwatchers/filelog/log_watcher.go]: Nice catch! :)

Comment (yuzhiquan) [reply pkg/systemlogmonitor/logwatchers/filelog/log_watcher.go]: > Nice catch! :)

thanks, i am a beginner for k8s.

**Pull request merged**